apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx

```diff
@@ -21,6 +22,7 @@ export interface PageWrapperProps {
   menu?: ReactNode
   children?: ReactNode
   authUser?: AuthUser
+  shouldUseHistory?: boolean
```

Comment:
Can we rename to be clearer?
```suggestion
  backHrefHistory?: boolean
```